app/controllers/user_controller.rb

```diff
@@ -0,0 +1,62 @@
+class UserController < ApplicationController
+  include AuthenticatedApiConcern
+
+  HOMEPAGE_ATTRIBUTES = %w[email email_verified transition_checker_state].freeze
```

Comment:
Could these be symbols (mainly for consistency in the `attributes.dig` arguments), or do we need them to be strings?